File: Codewars/8kyu/8kyu_counting_sheep/src/main.py
def count_sheeps(arrayOfSheeps):
    return len([sheep for sheep in arrayOfSheeps if sheep])

array1 = [True,  True,  True,  False,
          True,  True,  True,  True,
          True,  False, True,  False,
          True,  False, False, True,
          True,  True,  True,  True,
          False, False, True,  True]

print count_sheeps(array1) == 17

File: Codewars/8kyu/8kyu_counting_sheep/src/main.js
function mysolution(arrayOfSheep) {
    cnt = 0
    arrayOfSheep.forEach(val => {
        if(val) cnt++
    })
    return cnt
}

function othersolution(arrayOfSheep) {
    return arrayOfSheep.filter(Boolean).length
}

count_sheeps = mysolution

var array1 = [true,  true,  true,  false,
    true,  true,  true,  true ,
    true,  false, true,  false,
    true,  false, false, true ,
    true,  true,  true,  true ,
    false, false, true,  true ];

console.log(count_sheeps(array1) == 17)